cli/src/main/scala-2.11/coursier/cli/Fetch.scala

```diff
@@ -7,7 +7,7 @@ import caseapp._
 
 import scala.language.reflectiveCalls
 
-final case class Fetch(
+case class Fetch(
```

Comment:
removed 'final' so I can create an instance of it in test. please advise if there's a better way.